src/main.rs

```diff
@@ -76,6 +77,39 @@ fn run() -> Result<(), failure::Error> {
                 .subcommand(
                     SubCommand::with_name("list")
                 )
+                .subcommand(
+                    SubCommand::with_name("write")
+                        .arg(
+                            Arg::with_name("expiration")
+                            .short("e")
+                            .long("expiration")
+                            .takes_value(true)
+                            .value_name("SECONDS")
+                            .help("the time, measured in number of seconds since the UNIX epoch, at which the entries should expire"),
+                        )
+                        .arg(
+                            Arg::with_name("time-to-live")
```

Comment:
We should consider using one of many duration parsing [libraries](http://tailhook.github.io/humantime/humantime/struct.Duration.html) for this, unlike cache, KV typically has much longer expiration. We don't want people getting out their calculators.

By default, numbers are interpreted as seconds, but now I can do things like `24hr 30m`. Very convenient, and maybe there are other places in the codebase that could benefit from it.

Reply:
We're actually removing this from the commandline in favor of having folks specify this in their KV upload json file, but the comment still stands--
I think this is one case where providing functionality on top of the preexisting KV API can be nice (https://api.cloudflare.com/#workers-kv-namespace-write-multiple-key-value-pairs). While using an epoch time converter isn't the worst, it definitely is something that can be taken out of this upload process :)

Reply:
i would advise users to generate this json, not write it by hand. that really makes this parameter make more sense as seconds.

Reply:
^ Agreed.